multiproof_test.go

```diff
@@ -37,14 +37,14 @@ func TestMultiProofWithIndices(t *testing.T) {
 
 			// Test proof for all combinations of data.
 			var proof *MultiProof
-			combinations := 1<<len(test.data) - 1
+			combinations := 1<<len(tree.Data) - 1
```

Comment:
There are other tests that use `test.data` in a similar fashion, please could you update those as well?